### PR TITLE
* add setup for CircleCI

### PR DESCRIPTION
**Pull request by klokane**

### CODE REVIEW

**Comment by pksunkara** [comment]
Please add circleCI svg image on the README. Until then this is blocked.

**Comment by klokane** [comment]
OK, I'll push it in a few day while update doc for drafter

**Comment by zdne** [comment]
@pksunkara please move the internal tracking card accordingly (if you review and merge / block a PR)

**Comment by klokane** [comment]
@pksunkara icon added

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # drafter
 Snowcrash parser harness
+
+[![Circle CI](https://circleci.com/gh/apiaryio/drafter.svg?style=svg&circle-token=e9446ed7fa2c180888e9fde172cf4cd6f57479f4)](https://circleci.com/gh/apiaryio/drafter)
```
Comment (pksunkara) [inline README.md]: We don't need the token here. And the badge should be for only the master branch.

Comment (pksunkara) [inline README.md]: And this badge should be either on the header line `# drafter` or just after it.

*I prefer on the header line*

Comment (klokane) [inline README.md]: I look for doc, it seems CirceCI prefer tokens https://circleci.com/docs/status-badges

Comment (pksunkara) [inline README.md]: Those tokens are for projects which are private.

**Comment by klokane** [comment]
And now? :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
-# drafter
+# drafter [![Circle CI](https://circleci.com/gh/apiaryio/drafter.svg?style=shield)](https://circleci.com/gh/apiaryio/drafter)
```
Comment (pksunkara) [inline README.md]:
```
https://circleci.com/gh/apiaryio/drafter/tree/master.svg
```

We need to show only the master branch.

**Pull request merged**